README.md

```diff
@@ -33,7 +33,10 @@ This project uses [Yarn](https://yarnpkg.com/) for package management. Yarn help
 
 1. `yarn install`
    - This will also run [Lerna](https://lerna.js.org/) `bootstrap` which allows us to have multiple packages within the same repo (a monorepo). Lerna installs all our dependencies and symlinks any cross-dependencies.
-1. `yarn start`
+2. `yarn build`
```

Comment:
I kinda feel like we should just remove this 'getting started' section entirely, the Scripts section right after it details all the scripts.

Reply:
Agreed, with e7504d5 work already underway (which is what prompted this change) `install` and `start` would get folks up and running first time around.

Reply:
We probably need to update the entry for `yarn start`, though, to explain that `yarn build` is a prerequisite step

Reply:
And we could add `yarn install` at the very top and move `yarn start` after `yarn build`